### ISSUE TITLE
wkhtmltopdf not generating accessible PDFs

### ISSUE THREAD

**Issue by ghost** [wkhtmltopdf/wkhtmltopdf#1616]
Hi all, using newly-upgraded 0.12.0 64-bit on a Windows Server 2008 R2 with IIS. (was an issue with previous versions as well).

wkhtmltopdf ignores certain elements that are required to make WCAG 2.0 compliant PDFs. For example, a generated PDF will be missing alternate text for images, as well as the page language, even though those are explicitly set in the HTML source page.

As we are legislated to provide accessible content on our website, whether this is addressed will dictate whether we are able to continue using this software. Thank you!
